services/http/api/main.js

```diff
@@ -78,6 +77,16 @@ app.post('/measurement', async (req, res) => {
     }
 });
 
+app.post('/survey', (req, res) => {
+    res.status(200).send({
+        message: 'Received'
+    })
+});
+
+app.get('/', (req, res) => {
+    return res.status(200).end();
+});
+
```

Comment:
We should add specification for these endpoints in the API reference. I can tell you more about this in a meeting.

Reply:
Sounds like a great idea? When would be available to having this meeting?